cmd/ui/main.go

```diff
@@ -2,49 +2,51 @@ package main
 
 import (
 	"embed"
-	"encoding/json"
+	"flag"
 	"io/fs"
 	"net/http"
+	"net/http/httputil"
+	"net/url"
 	"os"
 	"path/filepath"
 
 	"github.com/sirupsen/logrus"
 )
 
+var log = logrus.New()
+
 func main() {
-	log := logrus.New()
+	apiUrlFlag := flag.String("api-url", "http://localhost:8000", "The URL for the applications API server")
+
 	mux := http.NewServeMux()
 
 	mux.Handle("/health/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusOK)
```

Comment:
You don't need to do this, it's the default value for responses if there's no other status.

https://golang.org/src/net/http/server.go#L1600

Reply:
Strange. So what is the second argument to `mux.Handle`?

Reply:
Sorry, you don't need to `w.WriteHeader(http.StatusOK)`

Reply:
Just an empty handler func then?